examples/exporting_for_inference.ipynb

```diff
@@ -0,0 +1,329 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7b755249",
+   "metadata": {},
+   "source": [
+    "# 🥡 Exporting for Inference\n",
+    "\n",
+    "Composer models are also `torch.nn.Module`, and thus can be exported like any other PyTorch module. In this tutorial, we walk through how to export your models into various common formats: [ONNX](https://onnx.ai/), [TorchScript](https://pytorch.org/docs/stable/jit.html), and [torch.fx](https://pytorch.org/docs/stable/fx.html). For more detailed options and configuration settings, please consult the linked documentation.\n",
+    "\n",
+    "## Algorithm compatibility\n",
+    "Some of our algorithms alter the model architecture in ways that may render them incompatible with some of the export procedures above. For example, BlurPool replaces some instances of `Conv2d` with `BlurConv2d` layers which are not compatible with `torch.fx` as they have data-dependant codepath. \n",
+    "\n",
+    "The following table shows which algorithms are compatible with which export formats for inference.\n",
+    "\n",
+    "|                        | torchscript | torch.fx | ONNX |\n",
+    "|------------------------|-------------|----------|------|\n",
+    "| apply_blurpool         | X           |          | X    |\n",
```

Comment:
Can we do check marks and crosses in this table? Currently crosses come across as algorithms not working with the corresponding export format.

Reply:
Done!